modules/admin_manual/pages/configuration/server/security/hsmdaemon/index.adoc

```diff
@@ -19,9 +19,7 @@ The hsmdaemon is a daemon, provided by ownCloud, to delegate encryption to an {h
 This can be necessary, as PHP cannot, directly, interface with {pkcs11-url}[a PKCS11 stack]; neither with an API wrapper, because one does not exist, nor via the OpenSSL bindings.
 Because of this, a separate process is needed to decrypt anything with the private key stored in an HSM.
 
-When using hsmdaemon with an HSM, the keys are not stored on the same physical machine as ownCloud. 
-As a result, if someone steals the machine that stores your ownCloud instance and boots it outside your network, your data cannot be accessed. 
-As the HSM is not accessible, the keys cannot be read, so they cannot be used to decrypt your data.
+NOTE: When using hsmdaemon with an HSM, the keys _can_ still be stored on the same physical machine as ownCloud.
```

Comment:
Maybe I have a wrong understanding of the meaning `can` implies. To me, `can` carries a tone of *hey this cool thing CAN be done*, a positve connotation. In this case, keys being staying on the same machine is a bad thing IMO. Which is why I would like to express more of a *there might still be a problem left*. Something like:
`NOTE: When using hsmdaemon with an HSM, the keys _might_ still be stored on the same physical machine as ownCloud, depending on the HSM implementation.`

But it depends on what we are trying to say in general:
- These are the threat models you can cover with this config, or
- This is how you configure it, but be aware that the following threat models are not covered.

The latter would mean coming up with an endless list of uncovered things. The former means stating a finite number of threat models, maybe ordered by degree of security and providing a config for that. Listing the same config over and over seems redundant, so maybe providing the config steps that allow the admin to reach the minimum degree of security (internal encryption) with added/changed config steps needed to reach the next higher level of security would be a cool thing. Like *congratulations, you are now protected against x*, or in gamification terms *bronze/silver/gold/platinum/... encryption unlocked* ...